src/plugins/intel_gpu/src/plugin/ops/reduce.cpp

```diff
@@ -123,10 +126,10 @@ static void CreateReduceOp(Program& p, const std::shared_ptr<ngraph::Node>& op,
                                            cldnn::reorder_mean_mode::subtract,
                                            op->get_friendly_name());
         p.AddPrimitive(reorder_prim);
-        p.AddPrimitiveToProfiler(op, reorderLayerName);
-    } else {
-        p.AddPrimitiveToProfiler(op);
+        resultLayerName = reorderLayerName;
     }
+
```

Comment:
Please add reduce primitive to AddInnerPrimitiveToProfiler somewhre

Reply:
Fixed